modules/project/README.md

```diff
@@ -344,12 +388,66 @@ module "service-project" {
   ]
   shared_vpc_service_config = {
     host_project = module.host-project.project_id
-    service_identity_subnet_iam = {
+    service_identity_subnets_iam = {
       "europe-west1/gce" = ["compute"]
     }
+    subnets_iam = {
+      "europe-west1/gce" = ["group:team-1@example.com"]
+    }
+  }
+}
+# tftest modules=2 resources=7 inventory=shared-vpc-subnet-grants.yaml
+```
+
+The last example provides a reference Shared VPC configuration that adheres to the principle of least privilege by:
+- Granting API services and other identities roles at the subnet level
```

Comment:
I would just show the org policy in the project role example above and scrap this example, combining the two does not make a lot of sense IMHO.

Reply:
I added this last example just for showing a "reference" configuration for Shared VPC with permissions assigned at subnet level + org policy which might be a best practise, WDYT?

Reply:
you don't need subnet-level with org policy IMHO :) I find this confusing, let's merge the org policy where we show project-level IAM